packages/eslint-config/.eslintrc.js

```diff
@@ -81,14 +79,14 @@ module.exports = {
     "no-bitwise": "error",
     "no-caller": "error",
     "no-eval": "error",
-    "no-shadow": "error",
+    "no-shadow": "off",
```

Comment:
oooh, we shouldn't turn things off this will affect Jobber and other repos that uses this package

Reply:
It was also disabled in jobber-mobile: https://github.com/GetJobber/jobber-mobile/pull/3355/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5

It's handled by `@typescript-eslint/no-shadow`: https://github.com/GetJobber/atlantis/blob/JOB-63536-bump-eslint/packages/eslint-config/.eslintrc.js#L72